backends/p4tools/common/lib/variables.h

```diff
@@ -7,7 +7,12 @@
 /// Variables internal to P4Tools. These variables do not exist in the P4
 /// program itself, but are generated and added to the environment by the P4Tools tooling. These
 /// variables are also used for SMT solvers as symbolic variables.
-namespace P4Tools::ToolsVariables {
+namespace P4Tools {
+
+/// A list of constraints containing symbolic variables.
```

Comment:
Can we say a word about the semantics about such a vector, at least at an intuitive level?